client/app/dedicatedCloud/service-pack/translations/Messages_fr_FR.json

```diff
@@ -1,20 +1,29 @@
 
-  "dedicatedCloudDashboardTilesOptionsServicePack_name_nsx-and-vrops": "VMware Network Virtualization (NSX) et VMware vRealize Operations (vROps)",
+  "dedicatedCloudDashboardTilesOptionsServicePack_displayName_nsx-and-vrops": "VMware Network Virtualization (NSX) et VMware vRealize Operations (vROps)",
+  "dedicatedCloudDashboardTilesOptionsServicePack_fullDisplayName_nsx-and-vrops": "VMware Network Virtualization (NSX) et VMware vRealize Operations (vROps)",
   "dedicatedCloudDashboardTilesOptionsServicePack_description_nsx-and-vrops": "<p>Avec NSX, configurez les règles d’accès à vos réseaux, créez votre politique de sécurité et déployez en quelques minutes les différents services réseau nécessaires à la construction de votre infrastructure.</p><p>Grâce à vROps, pilotez votre infrastructure, activez le monitoring de vos machines virtuelles, détectez toutes les anomalies et anticipez facilement vos besoins en ressources pour assurer la qualité de service (QoS) de votre plateforme.</p>",
 
-  "dedicatedCloudDashboardTilesOptionsServicePack_name_pcidss": "Payment Card Industry Data Security Standard (PCI DSS)",
+  "dedicatedCloudDashboardTilesOptionsServicePack_displayName_pcidss": "Payment Card Industry Data Security Standard (PCI DSS)",
+  "dedicatedCloudDashboardTilesOptionsServicePack_fullDisplayName_pcidss": "Certification Payment Card Industry Data Security Standard (PCI DSS)",
   "dedicatedCloudDashboardTilesOptionsServicePack_description_pcidss": "Les données des cartes de paiement sont la cible de nombreuses tentatives de piratage et de fraudes. Grâce à PCI DSS, OVH a mis en place les mesures de sécurité les plus poussées en la matière pour assurer le meilleur niveau de sécurité pour vos infrastructures de paiement.",
-  "dedicatedCloudDashboardTilesOptionsServicePack_name_hds": "Hébergement des données de santé (HDS)",
+
+  "dedicatedCloudDashboardTilesOptionsServicePack_displayName_hds": "Hébergement des données de santé (HDS)",
+  "dedicatedCloudDashboardTilesOptionsServicePack_fullDisplayName_hds": "Certification Hébergement des données de santé (HDS)",
   "dedicatedCloudDashboardTilesOptionsServicePack_description_hds": "Les données et systèmes d’information de santé constituent un élément essentiel de la vie privée des patients. Conscient de ces enjeux, OVH a mis tout son savoir-faire et son expertise en matière de sécurité pour proposer une offre d’hébergement des données de santé (HDS), respectueuse des standards de sécurité les plus exigeants du domaine.",
-  "dedicatedCloudDashboardTilesOptionsServicePack_name_hipaa": "Health Insurance Portability and Accountability Act (HIPAA)",
+
+  "dedicatedCloudDashboardTilesOptionsServicePack_displayName_hipaa": "Health Insurance Portability and Accountability Act (HIPAA)",
+  "dedicatedCloudDashboardTilesOptionsServicePack_fullDisplayName_hipaa": "Certification Health Insurance Portability and Accountability Act (HIPAA)",
```

Comment:
Some part on this file aren't supposed to be translated right?

I know it was already the case for some previous terms but I think a better solution would be to extract them into a regular constant.

Do you agree?